standards/SPIFFE-ID.md

```diff
@@ -45,6 +46,15 @@ Trust domain names are nominally self-registered, unlike public DNS there is no
 
 The trust domain name is defined as the authority component of the URI - specifically, the `host` part of the authority. The `userinfo` and `port` parts of the authority component MUST NOT be set, and the `:` delimiter MUST NOT be present. Please see section 3.2 of [RFC 3986](https://tools.ietf.org/html/rfc3986) for more information.
 
+#### 2.1.1. Trust Domain Name Collisions
+
+Trust domain names are self-registered, ie. trust domain operators are free to choose any trust domain name they find suitable. Since there is no centralized authority for regulation or registration of trust domain names, there is no guarantee of global uniqueness nor is there any technical means for preventing distinct trust domains from using identical trust domain names.
```

Comment:
```suggestion
Trust domain operators are free to choose any trust domain name they find suitable: there is no centralized authority for regulation or registration of trust domain names. Thus, there is no guarantee of global uniqueness nor is there any technical means for preventing distinct trust domains from using identical trust domain names.
```

"self-registration" normally means there is a central authority, but the registration process is automated or unattended, like a sign-up sheet posted on a bulletin board, or creating a Twitter account.  Trust domain names aren't like that, so I've dropped that phrase and reworked things a bit. PTAL